src/js/ui/LottoResultComponent.js

```diff
@@ -0,0 +1,29 @@
+import { winningForm } from '../constants.js';
+import { $profit, $tbody } from '../dom.js';
+
+export class LottoResultComponent {
+  lottoResultModel;
+
+  constructor(lottoResultModel) {
+    this.lottoResultModel = lottoResultModel;
+    this.render();
+  }
+
+  render() {
+    $tbody.replaceChildren();
+
+    const result = Object.keys(winningForm)
+      .map(
+        (place) => `
+          <tr class="text-center">
+            <td class="p-3">${winningForm[place].number}</td>
+            <td class="p-3">${winningForm[place].winnings}</td>
+            <td class="p-3">${this.lottoResultModel.winningResult[place]}개</td>
+          </tr>`
+      )
+      .join('');
+
+    $tbody.insertAdjacentHTML('afterbegin', result);
+    $profit.textContent = this.lottoResultModel.profitRate;
+  }
```

Comment:
```suggestion
export class LottoResultComponent {
  constructor(resultModel) {
    this.render(resultModel);
  }

  render(resultModel) {
    $tbody.replaceChildren();

    const result = Object.keys(winningForm)
      .map(
        (place) => `
          <tr class="text-center">
            <td class="p-3">${winningForm[place].number}</td>
            <td class="p-3">${winningForm[place].winnings}</td>
            <td class="p-3">${resultModel.winningResult[place]}개</td>
          </tr>`
      )
      .join('');

    $tbody.insertAdjacentHTML('afterbegin', result);
    $profit.textContent = resultModel.profitRate;
  }
```

멤버 변수가 없어도 되지 않을까요?

---

멤버변수가 있으면 좋은 점 vs 안 좋은 점
고민해보시면 좋을 것 같아요!
이렇게 단순하게 인자로 받아서 쓰는건 반대로 어떤 장점이 있을까요?

Reply:
멤버변수가 있으면 외부에서 언제든지 사용할 수 있지만, 지금 LottoResultComponent의 멤버변수를 사용할 필요가 없기 때문에 리뷰어님 코멘트대로 제거하면 될 것 같습니다.